### PR TITLE
Fix broken external links

### PR DESCRIPTION
**Pull request by Eric-Arellano**

### CODE REVIEW

File: scripts/lib/linkCheckerIgnores.ts [hunk 1]
```diff
@@ -22,19 +22,6 @@ type FilesToIgnores = { [id: string]: string[] };
 const SHOULD_BE_FIXED: FilesToIgnores = {
   "docs/api/qiskit-ibm-provider/ibm-provider.md": ["ibm_provider"],
   "docs/api/qiskit-ibm-runtime/ibm-runtime.md": ["runtime_service"],
-  "docs/api/qiskit/pulse.md": [
```
Comment (Eric-Arellano) [inline scripts/lib/linkCheckerIgnores.ts:25]: Fixed thanks to @arnaucasau!

File: scripts/lib/linkCheckerIgnores.ts [hunk 2]
```diff
@@ -58,26 +45,27 @@ const SHOULD_BE_FIXED: FilesToIgnores = {
...
-  ],
   "docs/api/qiskit/utils.md": [
     "https://github.com/python-constraint/python-constraint%3E__",
     "#qiskit.utils.optionals.HAS_TESTTOOLS",
     "#qiskit.utils.optionals.HAS_GRAPHVIZ",
     "#qiskit.utils.optionals.HAS_PYDOT",
   ],
   "docs/api/qiskit-ibm-provider/qiskit_ibm_provider.IBMProvider.md": [
-    "https://auth.quantum.ibm.com/api",
+    "https://auth.quantum-computing.ibm.com/api",
```
Comment (Eric-Arellano) [inline scripts/lib/linkCheckerIgnores.ts:55]: This was a bad change - the API docs still use `quantum-computing` despite the URL migration.

File: scripts/lib/linkCheckerIgnores.ts [hunk 3]
```diff
@@ -58,26 +45,27 @@ const SHOULD_BE_FIXED: FilesToIgnores = {
...
+    "https://docs.python-requests.org/en/latest/api/",
```
Comment (Eric-Arellano) [inline scripts/lib/linkCheckerIgnores.ts:56]: New issue. I'll open a PR to fix this

File: scripts/lib/linkCheckerIgnores.ts [hunk 4]
```diff
@@ -58,26 +45,27 @@ const SHOULD_BE_FIXED: FilesToIgnores = {
...
   ],
   "docs/api/qiskit/algorithms.md": ["https://www.qiskit.org/terra"],
   "docs/api/qiskit/qiskit.algorithms.Grover.md": [
     "https://qiskit.org/textbook/ch-algorithms/grover.html",
   ],
+  "docs/api/qiskit-ibm-provider/release-notes.md": [
+    "https://github.com/Qiskit/qiskit-ibm-provider/blob/main/docs/tutorials/Migration_Guide_from_qiskit-ibmq-provider.ipynb",
```
Comment (Eric-Arellano) [inline scripts/lib/linkCheckerIgnores.ts:67]: New issue. I'll open a PR to fix this.

File: scripts/lib/linkCheckerIgnores.ts [hunk 5]
```diff
@@ -89,6 +77,9 @@ const EXPECTED: FilesToIgnores = {
   "docs/api/qiskit/qiskit.algorithms.optimizers.SPSA.md": [
     "https://ieeexplore.ieee.org/document/657661",
   ],
+  "docs/api/qiskit/synthesis.md": [
+    "https://www.cs.tau.ac.il/~nogaa/PDFS/r.pdf",
```
Comment (Eric-Arellano) [inline scripts/lib/linkCheckerIgnores.ts:81]: The link is valid. It's just slow to load.

**Review by arnaucasau — APPROVED** [review]
Looks good. Thank you for fixing the broken links Eric!

**Pull request merged**